Mathlib/Probability/Kernel/Composition/Basic.lean

```diff
@@ -1267,6 +1288,26 @@ lemma comp_id (κ : Kernel α β) : κ ∘ₖ Kernel.id = κ := by
 lemma id_comp (κ : Kernel α β) : Kernel.id ∘ₖ κ = κ := by
   rw [Kernel.id, deterministic_comp_eq_map, map_id]
 
+lemma id_map {f : α → β} (hf : Measurable f) : Kernel.id.map f = deterministic f hf := by
+  rw [← deterministic_comp_eq_map, comp_id]
+
+lemma id_comap {f : α → β} (hf : Measurable f) : Kernel.id.comap f hf = deterministic f hf := by
+  rw [← comp_deterministic_eq_comap, id_comp]
+
+lemma id_map_eq_id_comap {f : α → β} (hf : Measurable f) :
+    Kernel.id.map f = Kernel.id.comap f hf := by
+  rw [id_comap, ← id_map]
```

Comment:
That does not seem very useful, especially if the two lemmas above become simp lemmas.